IoT_master/tools/camera_initialize.py

```diff
@@ -0,0 +1,37 @@
+import cv2
+
+class Camera:
+    def __init__(self):
+        self.login = 'admin'
+        self.password = 'qwerty123456'
+        self.ip_address = '192.168.1.108'
+        self.channel = '1'
+        self.adres_part_1 = 'rtsp://'+self.login+':'+self.password+'@'+self.ip_address+'/cam/realmonitor?channel='
+        self.adres_part_2 = '&subtype=0'
+        
+    def camera(self,channel='thermal'):
+        if channel == 'thermal':
+            self.adres = self.adres_part_1+'2'+self.adres_part_2
+        else:
+            self.adres = self.adres_part_1+'1'+self.adres_part_2
+        self.cam = cv2.VideoCapture(self.adres)
+        self.ret, self.img = self.cam.read()
+        return self.ret, self.img
+
+if __name__=='__main__':
```

Comment:
Can you please add docstring that will provide some info of what is going on when you run this script?
I know what it does, but it’s be helpful for other people

Reply:
Done.